central/hash/manager/deduper.go

```diff
@@ -77,25 +86,34 @@ func skipDedupe(msg *central.MsgFromSensor) bool {
 	if !ok {
 		return true
 	}
-	if eventMsg.Event.GetProcessIndicator() != nil {
-		return true
-	}
-	// Autogenerated image integrations currently do not have consistent ID
-	if eventMsg.Event.GetImageIntegration() != nil {
-		return true
-	}
-	if alert.IsRuntimeAlertResult(msg.GetEvent().GetAlertResults()) {
-		return true
-	}
-	// This can occur for a very short-lived deployment where alerts are not generated
-	// but the deployment is being removed
-	if alert.IsAlertResultResolved(msg.GetEvent().GetAlertResults()) {
-		return true
-	}
-	if alert.AnyAttemptedAlert(msg.GetEvent().GetAlertResults().GetAlerts()...) {
-		return true
-	}
-	if eventMsg.Event.GetReprocessDeployment() != nil {
+	event := eventMsg.Event
+
+	switch event.Resource.(type) {
```

Comment:
Am I reading this correctly in that the items previously returning true via the series of ifs will now fall through to the default case and return true?

Reply:
This is correct. Basically, we are explicitly saying we want these objects. Let me refactor this so it's clear

Reply:
Does the recent change make it more clear?

Reply:
It does in my view, at least.